spec/unit/yardstick/ordered_set/left_shift_operator_spec.rb

```diff
@@ -13,4 +13,12 @@
   it { should include(item) }
 
   its(:length) { should be(1) }
+
+  context 'when operation is repeated' do
+    subject do
+      -> { set << item << item }
```

Comment:
I don't know if I've seen this specifically. Can it just work if you use `subject { set << item << item }` ?